mne/minimum_norm/inverse.py

```diff
@@ -1212,6 +1211,170 @@ def apply_inverse_epochs(epochs, inverse_operator, lambda2, method="dSPM",
+    orientations for a given location by a single value.
+    Using ``force_equal=True`` mimics this behavior by modifying the iterative
+    algorithm to choose uniform weights (equivalent to a 3x3 diagonal matrix
+    with equal entries).
+
+    It is necessary to use ``force_equal=True``
+    with loose orientation inverses (e.g., ``loose=0.2``), otherwise the
+    solution resembles a free-orientation inverse (``loose=1.0``).
+    It is thus recommended to use ``force_equal=True`` for loose orientation
+    and ``force_equal=False`` for free orientation inverses. This is the
+    behavior used when the parameter ``force_equal=None`` (default behavior).
+
+    References
+    ----------
+    .. [1] Hämäläinen M S and Ilmoniemi R. Interpreting magnetic fields of
+           the brain: minimum norm estimates. Medical & Biological Engineering
+           & Computing, 32(1):35-42, 1994.
+    .. [2] Dale A, Liu A, Fischl B, Buckner R. (2000) Dynamic statistical
+           parametric mapping: combining fMRI and MEG for high-resolution
+           imaging of cortical activity. Neuron, 26:55-67.
+    .. [3] Pascual-Marqui RD (2002), Standardized low resolution brain
+           electromagnetic tomography (sLORETA): technical details. Methods
+           Find. Exp. Clin. Pharmacology, 24(D):5-12.
+    .. [4] Pascual-Marqui RD (2007). Discrete, 3D distributed, linear imaging
+           methods of electric neuronal activity. Part 1: exact, zero error
+           localization. arXiv:0710.3341
+    """
+    # Dummy evoked class
+    class FakeEvoked:
+        def __init__(self, _info):
+            self.info = _info
+
+    _check_reference(FakeEvoked(info), inverse_operator['info']['ch_names'])
+    _check_option('method', method, INVERSE_METHODS)
+    _check_ori(pick_ori, inverse_operator['source_ori'],
+               inverse_operator['src'])
+    #
+    #   Set up the inverse according to the parameters
+    #
+    _check_ch_names(inverse_operator, info)
+
+    inv = _check_or_prepare(inverse_operator, nave, lambda2, method,
+                            method_params, prepared)
+
+    #
+    #   Pick the correct channels from the data
+    #
+    sel = _pick_channels_inverse_operator(cov['names'], inv)
+    logger.info('Applying inverse operator to cov...')
+    logger.info('    Picked %d channels from the data' % len(sel))
+    logger.info('    Computing inverse...')
+
+    K, noise_norm, vertno, source_nn = _assemble_kernel(inv, label, method,
+                                                        pick_ori)
+
+    # apply imaging kernel
+    sol = np.einsum('ij,ij->i', K, (cov.data[sel][:, sel] @ K.T).T)[:, None]
+
+    is_free_ori = (inverse_operator['source_ori'] ==
+                   FIFF.FIFFV_MNE_FREE_ORI and pick_ori != 'normal')
+
+    # always combine vector components
+    if is_free_ori:
+        logger.info('    Combining the current components...')
+        sol = sol[0::3] + sol[1::3] + sol[2::3]
```

Comment:
So what happens when the user specifies `pick_ori='vector'`?

Reply:
I guess I don't understand your question. If `pick_ori='vector'` then `_is_free_ori` is `True` and you combine the outputs from the inverse. 

What happens with fixed orientation inverse operators is unfortunately not tested since you can only use `pick_ori=None` with those and I don't have a comparable call to `apply_inverse_raw` that will facilitate testing. Presumably, this would be the same as using `pick_ori='normal'` and a suitably configured inverse.

Reply:
But `pick_ori='vector'` should produce a `VectorSourceEstimate`. We explicitly not support it in the `apply_lcmv_cov` and `apply_dics_cov` functions, as it should return vectors pointing in the direction of maximum power and we haven't gotten around to implementing that yet.